src/App.js

```diff
@@ -31,7 +31,7 @@ function App() {
               <Route path="profile" element={<Profile />} />
               <Route path="history" element={<History />} />
               <Route path="dashboard" element={<Dashboard />} />
-              <Route path="searchresult" element={<SearchResult />} />
+              <Route path="search" element={<SearchResult />} />
```

Comment:
We need fix this later. The 'SEARCH" link on the navbar should open the search form, not navigate to the search result page